system-addon/content-src/components/Search/Search.jsx

```diff
@@ -49,10 +59,11 @@ class Search extends React.Component {
       </label>
       <input
         id="newtab-search-text"
+        ref={input => { this.input = input; }}
```

Comment:
String refs are considered legacy, although i could add this function to the class definition to avoid re-binding it